src/data/all-moves.ts

```diff
@@ -3452,23 +3436,7 @@ export function initMoves() {
       MultiHitType._3,
     ),
     new AttackMove(Moves.MORTAL_SPIN, Type.POISON, MoveCategory.PHYSICAL, 30, 100, 15, 100, 0, 9)
-      .attr(
-        LapseBattlerTagAttr,
-        [
-          BattlerTagType.BIND,
-          BattlerTagType.WRAP,
-          BattlerTagType.FIRE_SPIN,
-          BattlerTagType.WHIRLPOOL,
-          BattlerTagType.CLAMP,
-          BattlerTagType.SAND_TOMB,
-          BattlerTagType.MAGMA_STORM,
-          BattlerTagType.SNAP_TRAP,
-          BattlerTagType.THUNDER_CAGE,
-          BattlerTagType.SEEDED,
-          BattlerTagType.INFESTATION,
-        ],
-        true,
-      )
+      .attr(LapseBattlerTagAttr, rapidSpinRemoveTags, true)
```

Comment:
```suggestion
      .attr(RemoveBattlerTagAttr, rapidSpinRemoveTags, true)
```

LapseBattlerTagAttr actually does this instead:

https://github.com/user-attachments/assets/3862bebc-19f6-4e2e-a942-7515cf85736a

Reply:
lol nobody noticed